src/Hl7.Fhir.Serialization/FhirXmlBuilderExtensions.cs

```diff
@@ -24,25 +28,58 @@ private static void writeTo(this XDocument doc, XmlWriter destination)
             destination.Flush();
         }
 
+        private static async Task writeToAsync(this XDocument doc, XmlWriter destination)
+        {
+            if (doc.Root != null)
+                doc.WriteTo(destination);
```

Comment:
```suggestion
                await doc.WriteToAsync(destination).ConfigureAwait(false);
```

Reply:
WriteToAsync is only available in netstandard2.0. We are supporting both netstandard1.3 and .NET Framework 4.5

Reply:
Available in netstandard2.1 actually

Reply:
Ah ok, I was not aware of that.